playlists/views.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+from django.shortcuts import redirect, render
+
+from playlists.forms import MakePlaylistForm
+
+
+def create_playlist(request):
+    form = MakePlaylistForm(user=request.user)
+    if request.method == 'POST':
+        form = MakePlaylistForm(
+            request.POST,
+            user=request.user,
+            token=request.session['token_response']['access_token'],
+        )
+        if form.is_valid():
+            form.save()
```

Comment:
When you have a moment, look into [messages](https://docs.djangoproject.com/en/4.0/ref/contrib/messages/#adding-a-message). Its useful to display one-time messages to users. It'll require updating your base template to consume the list of messages. But it's quite likely the correct way to alert the user that they did a thing successfully. It's quite possible that other actions in the app would benefit from using this framework.